### PR TITLE
Add dashboard module for chunks

### PR DESCRIPTION
**Pull request by sklirg**

### CODE REVIEW

**Comment by myth** [comment]
Looks good. Missing `DeleteView`? Also, about the directory placement. I havent really thought about it, but for now placing it under dashboard is probably OK. If we figure out some better way in the future it's no biggie to move it.

File: apps/dashboard/chunks/dashboard/views.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+class ChunkCreate(DashboardPermissionMixin, CreateView):
+    model = Chunk
+    fields = ('key', 'description', 'content')
+    permission_required = 'chunks.add_chunk'
+
+    def get_object(self, queryset=None):
+        # Overridden to work around https://github.com/django-guardian/django-guardian/issues/146,
+        # an issue with regards to guardian's PermissionMixin working with CreateView -
+        # where it requires an object even in a CreateView.
+        return None
```
Comment (cXhristian) [inline apps/dashboard/chunks/dashboard/views.py]: Maybe we should create some generic mixin for this?

**Comment by cXhristian** [comment]
Looks good. :shipit: 

Maybe we could use a better word than "chunks"? I'm not sure what that would be though.

File: apps/dashboard/chunks/dashboard/views.py (hunk 1 above)
Comment (sklirg) [inline apps/dashboard/chunks/dashboard/views.py]: We could voice in on the relevant issue and hope for a fix. :-) If we can fix it ourselves we can PR it as well.

**Comment by sklirg** [comment]
We already use the package named chunks, so I kept the name for consistency. If we want to replace django chunks (#1376) we can rename it at that time.

**Pull request merged**